libs/remix-solidity/src/index.ts

```diff
@@ -1,7 +1,7 @@
 export { Compiler } from './compiler/compiler'
 export { compile } from './compiler/compiler-helpers'
 export { default as CompilerInput, getValidLanguage } from './compiler/compiler-input'
-export { CompilerAbstract } from './compiler/compiler-abstract'
+export * from './compiler/compiler-abstract'
```

Comment:
This file only exports `CompilerAbstract ` class. is this change needed?

Reply:
yes we need to export everything, not only `CompilerAbstract`